test/parallel/test-readline-async-iterators.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const join = require('path').join;
+const readline = require('readline');
+const assert = require('assert');
+
+common.crashOnUnhandledRejection();
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const filename = join(tmpdir.path, 'test.txt');
+const file_content = 'abc123\n' +
+                     '南越国是前203年至前111年存在于岭南地区的一个国家\n';
+
+fs.writeFile(filename, file_content);
+
+async function tests() {
+  await (async function() {
+    const readable = fs.createReadStream(filename);
+    const rli = readline.createInterface({
+      input: readable,
+      crlfDelay: Infinity
+    });
+
+    let data = '';
+    for await (const k of rli) {
+      data += k;
+    }
```

Comment:
I think it would be nice to check how many iterations of the loop there were

Reply:
Good suggestion 👍